src/cdk-experimental/popover-edit/edit-ref.ts

```diff
@@ -23,6 +23,10 @@ export class EditRef<FormValue> implements OnDestroy {
   private readonly _finalValueSubject = new Subject<FormValue>();
   readonly finalValue: Observable<FormValue> = this._finalValueSubject.asObservable();
 
+  /** Emits when the user tabs out of this edit lens before closing. */
+  private readonly _tabbedOutSubject = new Subject<void>();
+  readonly tabbedOut: Observable<void> = this._tabbedOutSubject.asObservable();
```

Comment:
Does this have to be tab-specific? Would it be possible at all to use the existing `OverlayKeyboardDispatcher` for this?

Reply:
It's not really about the tab key specifically, but focus - when focus moves out of the lens form then this is triggered. Listening for the tab key is not sufficient as tab could be used to focus different elements within the form.

Would you prefer focusOutSubject?

Reply:
`blurSubject` would be good